guide/13-managing-arcgis-applications/hub-premium-guide.ipynb

```diff
@@ -0,0 +1,477 @@
+{
```

Comment:
<p>Change the syntax:</p><p> </p><p>"&gt; _Note:_ The pattern to add, search, get, update and delete Events is identical to that of Initiatives."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1448/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>